src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -72,8 +73,21 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $container->setParameter('sylius.resource.mapping', $config['mapping']);
         $container->setParameter('sylius.resource.settings', $config['settings']);
-        $container->setParameter('sylius.routing_path_bc_layer', $config['routing_path_bc_layer']);
+
+        $routingPathBcLayer = $config['routing_path_bc_layer'] ?? null;
+
+        if (null === $routingPathBcLayer) {
+            $routingPathBcLayer = class_exists(Transliterator::class);
+        }
+
+        if ($routingPathBcLayer && !class_exists(Transliterator::class)) {
```

Comment:
Cause Behat Transliterator is moved to optional packages (it's deprecated), using another system to urlize might change the routing path. That's why I decided to use the existing `routing_path_bc_layer` key that we introduced on this branch which has no release yet.